qt/lib/include/verovio-qt/veroviopage.h

```diff
@@ -0,0 +1,84 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        veroviopage.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __VEROVIOPAGE_H__
+#define __VEROVIOPAGE_H__
+
+//----------------------------------------------------------------------------
+
+#include <QQuickItem>
+#include <memory>
+
+//----------------------------------------------------------------------------
+
+#include "qtscenegraphdevicecontext.h"
+
+namespace vrv {
+class VerovioDoc;
+
+//----------------------------------------------------------------------------
+// VerovioPage
+//----------------------------------------------------------------------------
+
+/**
+ * This class represents one page of a Verovio document.
+ */
+class VerovioPage : public QQuickItem {
```

Comment:
I am not sure about the name either. It creates confusion with vrv::Page. It looks more like a View to me. So maybe QtVrvView?

Reply:
I got confused by view_page and page, so yet this class actually represents a vrv::View. Thus I renamed it to vrv_qt::View.